modules/thermal_hydraulics/test/tests/materials/ad_wall_heat_transfer_coefficient_schad/tests

```diff
@@ -0,0 +1,7 @@
+[Tests]
+  [test]
+    type = 'CSVDiff'
+    input = 'test.i'
```

Comment:
Which Peclet numebr are you getting? You will need to had a test for the other Peclet number conditions